### DISCUSSION TITLE
When using multiple models to generate answers, double-clicking on a word will cause the page to jump, making it impossible to select the word.

### DISCUSSION THREAD

**Discussion by jingjing2016** [open-webui/open-webui#7736] [General]
When using multiple models, there are some unfamiliar words in the generated answers that need to be double-clicked on the web page to be selected. However, when double-clicking, the web page content jumps upward, making it impossible to select. I wonder if it's just me.

**Comment by tjbck** [comment]
Screen recordings would definitely be helpful here.

**Reply by jingjing2016** [reply]
i am sorry,i can't .  try to Screen recordings ,  Sometimes the webpage will jump when double-clicking, and sometimes it won't.

**Comment by steveepreston** [comment]
Solved: https://github.com/open-webui/open-webui/commit/48637893b1f6ef95e00086d75a3d5ccf6d747acb
Details: https://github.com/open-webui/open-webui/issues/12939
